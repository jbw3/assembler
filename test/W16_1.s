# Test for W16

NOP # no operation
MOV R10, R15
NOT R14, R14

# test bases
addi r0, 0b10101
addi r1, 0B1001
addi r2, 0o56
addi r3, 0O12
addi r4, 100
addi r5, 254
addi r6, 0x81
addi r7, 0XF0
