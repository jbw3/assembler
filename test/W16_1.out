0000
01af
02ee
b015
b109
b22e
b30a
b464
b5fe
b681
b7f0
